src/main/java/it/uniroma3/mat/extendedset/intset/ConciseSetUtils.java

```diff
@@ -284,7 +283,7 @@ public static int getFlippedBit(int word)
 
   public static int flipBitAsBinaryString(int flipBit)
   {
-    return ((Number) Math.pow(2, flipBit)).intValue();
+    return 1 << flipBit;
```

Comment:
Behavior here is different for flipBit = 31, does that matter? It seems like the behavior of the new code makes _more_ sense: it actually flips the bit, where the old code flips all other bits. Judging by `MAX_LITERAL_LENGTH`, maybe bit 31 is never actually flipped, so the behavior doesn't matter.

@leventov does that sound right to you?